.golangci.yml

```diff
@@ -36,7 +36,7 @@ linters:
     #- gocritic
     #- gocyclo
     - gosimple
-    #- staticcheck
+    - staticcheck
```

Comment:
Could you please remove this? `staticcheck` is already present on line 26

Reply:
Sure, thanks for the comment. I removed this line.